qlty-config/src/config.rs

```diff
@@ -143,7 +143,6 @@ mod test {
     use crate::Workspace;
 
     #[test]
-    #[ignore] // always requires network connection
```

Comment:
No longer needs to be ignored since default source is now packaged as a crate and no longer requires a network call.